src/common/class.wpes-wp-post-field-builder.php

```diff
@@ -534,6 +534,127 @@ public function get_mappings( $args = array() ) {
 						),
 					),
 				),
+				'attachment' => array(
+					'type' => 'object',
+					'properties' => array(
+						'url' => array(
+							'type' => 'string',
+							'index' => 'not_analyzed',
+						),
+						// @question Can we create a custom analyzer that splits input into tokens only
+						// on the '/' character? Would that be a better solution than just creating
+						// three distinct mime-related fields?
+						'mime' => array(
```

Comment:
Yep. Let's create it as a multi_field where it gets split into a mime_type and a mime_type.raw field. The custom analyzer can just split on punctuation and lowercase everything. A variation on the lowercase analyzer: https://github.com/Automattic/wpes-lib/blob/master/src/common/class.wpes-analyzer-builder.php#L238